control/controlclient/direct.go

```diff
@@ -1297,7 +1298,8 @@ func initDevKnob() devKnobs {
 	}
 }
 
-var clockNow = time.Now
+var clock = &tstime.StdClock{}
+var clockNow = clock.Now // used instead of time.Now()
```

Comment:
Looking at the places that use `clockNow`, I think it would be preferable to pass a `tstime.Clock` to those `func`s as a parameter.